zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -231,6 +217,9 @@ pub struct ParametersBuilder {
     pre_nu6_funding_streams: FundingStreams,
     /// Post-NU6 funding streams for this network
     post_nu6_funding_streams: FundingStreams,
+    /// A flag indicating whether to allow changes to fields that affect
+    /// the funding stream address period.
+    should_lock_funding_stream_address_period: bool,
```

Comment:
This field and the related checks aren't strictly necessary if we want `ParametersBuilder` to be more flexible, it could be useful for debugging otherwise.